include/vcml/debugging/gdbserver.h

```diff
@@ -35,10 +35,19 @@ enum gdb_signal {
     GDBSIG_KILL = 9,
 };
 
+struct gdb_target {
```

Comment:
Can we move `gdb_target` inside `gdbserver`?

Reply:
done